client/pokemon/pokemon.ctrl.js

```diff
@@ -76,6 +76,16 @@ module.exports = class Pokemon {
   }
   parseAllProps () {
     this.parseBoxViewProps();
+    this.getRibbonClass = function(ribbon) {
```

Comment:
Can we make this a prototype method (i.e. part of the class) rather than redefining it for every instance?

e.g.

```js
class Pokemon {
  parseAllProps () {
    // ...other code, etc.
  }
  getRibbonClass (ribbon) {
    return ribbon.replace(foo).toLowerCase() // etc.
  }
}
```